README.md

```diff
@@ -96,9 +96,7 @@ Options:
 
 ```
 
-For examples, see LINK_TO_TURORIALS.
-
-<!-- TODO: Link to tutorials -->
+For examples, see [the tutorials section of our documentation](https://virtualship.readthedocs.io/en/latest/tutorials/index.html).
```

Comment:
Do we also want to enable virtualship.oceanparcels.org, and use that url here already?

Reply:
OK, also done on my side
<img width="1080" alt="Screenshot 2024-10-08 at 13 48 46" src="https://github.com/user-attachments/assets/2bf90d94-e1bc-40d5-ac54-b09689c85080">

May take a while for the DNS to become active?

Reply:
Looks good. i think we're good to merge